concent_api/concent_api/system_check.py

```diff
@@ -129,3 +153,27 @@ def storage_cluster_certificate_path_check(app_configs = None, **kwargs):  # pyl
         elif not os.path.isfile(certificate_path):
             errors.append(create_error_15_ssl_cert_path_is_not_a_file(certificate_path))
     return errors
+
+
+@register()
+def creation_new_chain_segment_time_check(app_configs  = None, **kwargs):  # pylint: disable=unused-argument
+    errors = []
+    creation_new_chain_segment_time = settings.CREATION_NEW_CHAIN_SEGMENT_TIME
+    if not isinstance(creation_new_chain_segment_time, int):
+        errors.append(create_error_15_if_new_chain_segment_time_not_integer())
+    elif not creation_new_chain_segment_time > 0:
+        errors.append(create_error_16_if_new_chain_segment_time_is_not_bigger_than_0())
+    return errors
+
+
+@register
+def geth_container_address_check(app_configs  = None, **kwargs):  # pylint: disable=unused-argument
+    if hasattr(settings, 'GETH_CONTAINER_ADDRESS'):
```

Comment:
Same here, please check at the beginning that it is even set, and it must be set if PAYMENT_BACKEND is set to sci

Reply:
Done